okta-jwt-sso/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { BrowserRouter as Router, Route, Link, Routes } from "react-router-dom";
+
+function App() {
+  const loginWithOkta = () => {
+    window.location.href = "http://localhost:3001/login";
+  };
+
+  return (
+    <Router>
+      <div>
+        <nav>
+          <ul>
+            <li>
+              <Link to="/">Start</Link>
+            </li>
+            <li>
+              <button onClick={loginWithOkta}>Login with Okta</button>
+            </li>
+          </ul>
+        </nav>
+
+        <Routes>
```

Comment:
nit: we seem to be using only a single route here, so we can remove the router for brevity.